Source/WebCore/accessibility/AccessibilityObject.h

```diff
@@ -123,6 +123,7 @@ class AccessibilityObject : public AXCoreObject {
     bool isMediaTimeline() const { return false; }
     virtual bool isSliderThumb() const { return false; }
     bool isLabel() const { return isAccessibilityLabelInstance() || labelForObjects().size(); }
+    virtual bool isRoot() const { return false; }
```

Comment:
Maybe this should have a comment. We will have to figure out what this means in the context of site isolation.

Reply:
Done in latest commit!